tests/spectron/deselect-minimize-on-close.spectron.js

```diff
@@ -0,0 +1,107 @@
+const Application = require('./spectronSetup');
+const { isMac } = require('../../js/utils/misc');
+const robot = require('robotjs');
+const WindowsAction = require('./spectronWindowsActions');
+
+let configPath;
+let app = new Application({
+    startTimeout: Application.getTimeOut(),
+    waitTimeout: Application.getTimeOut()
+});
+let wActions;
+describe('Verify by deselecting Minimize on Close', () => {
+
+    let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
+
+    beforeAll((done) => {
+        return app.startApplication().then((startedApp) => {
+            app = startedApp;
+            getConfigPath().then((config) => {
+                configPath = config;
+                done();
+            }).catch((err) => {
+                done.fail(new Error(`Unable to start application error: ${err}`));
+            });
+            wActions = new WindowsAction(app);
+        }).catch((err) => {
+            done.fail(new Error(`Unable to start application error: ${err}`));
+        });
+    });
+
+    function getConfigPath() {
+        return new Promise(function (resolve, reject) {
+            app.client.addCommand('getUserDataPath', function () {
+                return this.execute(function () {
+                    return require('electron').remote.app.getPath('userData');
+                })
+            });
+            app.client.getUserDataPath().then((userConfigPath) => {
+                resolve(userConfigPath.value)
+            }).catch((err) => {
+                reject(err);
+            });
+        });
+    }
+
+    afterAll((done) => {
+        if (app && app.isRunning()) {
+            jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+            app.client.getWindowCount().then((count) => {
+                if (count > 0) {
+                    app.stop().then(() => {
+                        done();
+                    }).catch((err) => {
+                        done();
+                    });
+                } else {
+                    done();
+                }
+            })
+        } else {
+            done();
+        }
+    });
+    it('should bring the app to top', () => {
```

Comment:
Consider to use startApplication in this PR: https://github.com/symphonyoss/SymphonyElectron/pull/430/files

Reply:
fixed